tokens/properties/components/OptionsMenu.json

```diff
@@ -49,6 +49,27 @@
 					"value" : "{background.color.base.active.value}"
 				}
 			},
+            "selected":{
```

Comment:
The spacing is wrong :D

Reply:
Spacing issues were fixed in a recent [commit](https://github.com/wmde/wikit/pull/288/commits/8c2d68fb26388c80fc2a12312ad2f955ab07ea93)